.eslintrc.js

```diff
@@ -25,5 +25,7 @@ module.exports = {
     "no-param-reassign": 1,
     "arrow-parens": [2, "as-needed"],
     "prefer-object-spread": 0,
+    "no-param-reassign": 0,
```

Comment:
This rule is set at line 25

Reply:
![image](https://user-images.githubusercontent.com/22051776/64596916-63fde300-d3e7-11e9-9030-76532faed405.png)
Hey, if you look at the above image you'd see that I changed this rule because it interferes with pre-existing, unrelated code.